messages/en.json

```
@@ -132,5 +133,30 @@
       "returnByDescription": "Select how long you would like to borrow the item for.",
       "submit": "Submit"
     }
+  },
+  "shiftSchedule": {
+    "title": "Shift Schedule",
+    "administratorMenu": {
+      "administratorMenu": "Administrator Menu",
```

```suggestion
      "label": "Administrator Menu",
```

This way you dont have to write AdministratorMenu.AdministratorMenu when retrieving the translation